app/components/ITunesCard/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ *
+ * ITunesCard
+ *
+ */
+
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import 'antd/dist/antd.css';
```

Comment:
we shouldn't have to import the antd css file here. 

if you're facing some styling issue please do let me know

Reply:
Oh okay actually `line 5` is a mistake. I was trying to solve an issue with anti carousel. 
Then forgot to delete that line. I will remove it for sure